differential_equations2.py

```diff
@@ -0,0 +1,40 @@
+#forward Euler method to numerically solve a logistic growth model
+#and then plots the solution using matplotlib
+from typing import Callable, List, Tuple
```

Comment:
For python versions greater than 3.8 you no longer need to import `List`, `Tuple`, `Dict` etc from typing. You can instead just use `list`, `tuple`, `dict` etc directly as type hints